lib/teiserver_web/controllers/api/spads_controller.ex

```diff
@@ -216,8 +217,6 @@ defmodule TeiserverWeb.API.SpadsController do
     end
   end
 
-  defp get_member_lobby(nil), do: nil
-
   @spec get_member_lobby(non_neg_integer()) :: T.lobby() | nil
```

Comment:
Wouldn't a saner fix be something like...
```suggestion
  @spec get_member_lobby(non_neg_integer() | nil) :: T.lobby() | nil
```

(This might then need moving to the main definiton, removed by this PR?)

Reply:
That won't work. I tried it. I also asked on Elixir forums:
https://elixirforum.com/t/pattern-can-never-match-the-type/64094/6

Basically dialyzer reckons that all the calls it found to this function do not use nil. If you search there's only one line that calls it
```
    target_id = int_parse(target_id_str)
    lobby = get_member_lobby(target_id)
```
and dialyzer reckons that will never be nil. Just to confirm, I asked on Elixir forums if deleting is the appropriate answer. We'll see.

Reply:
Oooh, okay, gotcha. Let's wait what the wise wizards reply :)